includes/class-rum.php

```
@@ -0,0 +1,320 @@
+			}
+
+			$token = isset( $params['token'] ) ? sanitize_text_field( (string) $params['token'] ) : '';
+			if ( ! self::is_valid_token( $token ) ) {
+				return array(
+					'ok'      => false,
+					'status'  => 401,
+					'message' => __( 'Invalid beacon token.', 'performance-optimisation' ),
+				);
+			}
+
+			$ip = isset( $_SERVER['REMOTE_ADDR'] ) ? sanitize_text_field( wp_unslash( $_SERVER['REMOTE_ADDR'] ) ) : '';
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $ip. Configured minimum length is 3. (undefined)

(ShortVariable)